src/LibHdb++Cassandra.cpp

```diff
@@ -1669,10 +1726,13 @@ void HdbPPCassandra::configure_Attr(string name,
 	//ID already present but different configuration (attribute type)
 	if(find_attr_result == FoundAttrWithDifferentType)
 	{
-		ERROR_MSG(facility << "/" << attr_name << " already configured with different configuration" << endl);
-		ERROR_MSG("Please contact your administrator for this special case" << endl);
-		// Need to contact an administrator in this case!?
-		// TODO (SJ) How is this handled?
+		stringstream error_desc;
+
+		error_desc << facility << "/" << attr_name << " already configured with different configuration."
```

Comment:
In general, I would rather use ends instead of endl at the end of the error descriptions in the thrown exceptions.